modules/bibformat/lib/elements/bfe_authority_publications.py

```diff
@@ -103,11 +102,14 @@ def format_element(bfo):
     "&sc=1" + \
     "&ln=" + bfo.lang
     prefix = prefix_pattern % url_str
```

Comment:
If there are no publications this variable is not bound, probably it should be inside the if too :wink: